db/seeders/items_seeder.rb

```diff
@@ -0,0 +1,23 @@
+# Creates the Base Items
+
+class ItemsSeeder
+  BASE_ITEMS = File.read(Rails.root.join("db", "base_items.json"))
+
+  def self.seed
+    items_by_category = JSON.parse(BASE_ITEMS)
```

Comment:
Another nit, but `BASE_ITEMS` is really `BASE_ITEMS_JSON_TEXT` since it's text and not an array. Why not just include the call to JSON.parse right there?:

   BASE_ITEMS = JSON.parse(File.read(Rails.root.join("db", "base_items.json")))

...and use that in the code below? (Possibly rename it `ITEMS_BY_CATEGORY` if you want.)

Reply:
Better